resources/ru/cemu.po

```diff
@@ -152,921 +1146,1090 @@ msgstr ""
+#: GeneralSettings2.cpp
+msgid "Are you sure you want to delete the account {} with id {:x}?"
+msgstr "Вы уверены, что хотите удалить аккаунт {} с id {:x}?"
+
+#: GeneralSettings2.cpp
+msgid "Confirmation"
+msgstr "Подтверждение"
+
+#: GeneralSettings2.cpp
+msgid "You have to restart the game in order to apply the new settings."
+msgstr "Вы должны перезапустить игру, чтобы применить новые настройки."
+
+#: GeneralSettings2.cpp MainWindow.cpp
+msgid "Information"
+msgstr "Информация"
 
-#: GeneralSettings2.cpp:776
+#: GeneralSettings2.cpp
 msgid "Select a directory containing games."
 msgstr "Выберите директорию, содержащую игры."
 
-#: GraphicPacksWindow2.cpp:41 graphicPacksWindow.cpp:36
-msgid "Graphic packs"
-msgstr "Графические пакеты"
+# stutter - лаг, притормаживание?
+#: gpu7ShaderCache.cpp
+msgid ""
+"Outdated shader cache\n"
+"\n"
+"The stored shader cache for this game was created with a Cemu version prior "
+"to 1.16.0.\n"
+"\n"
+"There are no significant downsides to reusing an existing cache but it may "
+"contain bloat that is no longer used by current versions of Cemu.\n"
+"\n"
+"It is highly recommended to start a new cache. Note that a new cache means "
+"that you will experience additional stutter during the first minutes of "
+"gameplay."
+msgstr ""
+"Устаревший кэш шейдеров\n"
+"\n"
+"Хранимый кэш шейдеров для этой игры был создан на Cemu версии до 1.16.0.\n"
+"\n"
+"Нет никаких существенных недостатков в повторном использовании существующего "
+"кэша, но он может содержать лишние включение, которые больше не используются "
+"текущими версиями Cemu.\n"
+"\n"
+"Настоятельно рекомендуется создать новый кэш. Обратите внимание, что новый "
+"кэш означает, что вы будете испытывать дополнительные притормаживания в "
+"течение первых минут игры."
 
-#: GraphicPacksWindow2.cpp:71
+#: gpu7ShaderCache.cpp
+msgid "Shader cache migration"
+msgstr "Перенос кэша шейдеров"
+
+#: gpu7ShaderCache.cpp
+msgid "Use existing cache"
+msgstr "Использовать существующий кэш"
+
+#: gpu7ShaderCache.cpp
+msgid "Create new cache [recommended]"
+msgstr "Создать новый кэш [рекомендуется]"
+
+#: GraphicPacksWindow2.cpp LoggingWindow.cpp toolMemorySearcher.cpp
+msgid "Filter"
+msgstr "Фильтр"
+
+#: GraphicPacksWindow2.cpp
+msgid "Installed games"
+msgstr "Установленные игры"
+
+#: GraphicPacksWindow2.cpp
 msgid "Graphic pack"
 msgstr "Графический пакет"
 
-#: GraphicPacksWindow2.cpp:81
-msgid "Active preset"
-msgstr "Активные предустановки"
-
-#: GraphicPacksWindow2.cpp:93
+#: GraphicPacksWindow2.cpp
 msgid "Description"
 msgstr "Описание"
 
-#: GraphicPacksWindow2.cpp:103
+#: GraphicPacksWindow2.cpp
 msgid "Control"
 msgstr "Управление"
 
-#: GraphicPacksWindow2.cpp:224
-msgid "This is an old graphic pack, therefore it has no description."
-msgstr "Это старый графический пакет, поэтому у него нет описания."
+#: GraphicPacksWindow2.cpp
+msgid "Download latest community graphic packs"
+msgstr "Загрузить новейшие пользовательские графические пакеты"
```

Comment:
Since this string is placed in a button, consider if there's a more compact translation that doesn't lose the meaning (if there is not, leave it as it is).